qi-lib/flow/core/compiler/deforest/syntax.rkt

```diff
@@ -29,12 +33,19 @@
          (for-syntax racket/base
                      syntax/parse))
 
+;; Literals set used for matching Fusable Stream Literals
```

Comment:
I'd prefer these comments showed up in the original commit introducing this
file. That might be challenging at this point, so a "document" commit is
acceptable.